src/main/java/de/clearit/kindergarten/NavigationsBtn.java

```diff
@@ -0,0 +1,134 @@
+package de.clearit.kindergarten;
+
+import java.awt.Dimension;
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+import javax.swing.JButton;
+import javax.swing.JPanel;
+
+public class NavigationsBtn extends JPanel implements ActionListener {
```

Comment:
Auch hier ist das kein *Button* mehr, sondern ein Panel -> Umbenennen in NavigationPanel z.B.